app/views/submissions/steps/trial/_step02.html.haml

```diff
@@ -1,17 +1,17 @@
 .alert.alert-info
   Before uploading your trait score sheet, please choose
-  %b plant traits
+  %b existing trait descriptors
   that were scored in the course of this trial,
   or define
   %b new trait descriptors.
   Further instructions are found below.
-  Examples of plant traits in the database are: 'oleic acid content' or ' seed mature time'
+  Examples of plant traits in the database are: 'oleic acid content' or ' seed mature time'.
 
 .form-group
   = content.label :trait_descriptor_list
-  - options = options_for_submission_select(@content.existing_trait_descriptors, :descriptor_name)
-  - options.concat options_for_submission_select(@content.new_trait_descriptors, :descriptor_name, id: false)
-  = content.select :trait_descriptor_list, options.html_safe, {},
+  -#- options = options_for_submission_select(@content.existing_trait_descriptors, :descriptor_name)
+  -#- options.concat options_for_submission_select(@content.new_trait_descriptors, :descriptor_name, id: false)
```

Comment:
These two lines are needed to show selected values when user returns to this step. Uncommenting fixes showing existing TDs, however it seems there is still an issue with new TDs (e.g. new TD is shown if there is also existing TD selected, but not if there is only new TD selected).